linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/ticket/RedisClient.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.linkis.server.ticket
+
+import org.apache.commons.lang3.StringUtils
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig
+import org.apache.linkis.common.conf.CommonVars
+import redis.clients.jedis.{Jedis, JedisPool, JedisSentinelPool}
+import scala.collection.JavaConverters.setAsJavaSetConverter
+
+object RedisClient {
+
+  private val RedisHost: String = CommonVars("linkis.redis.host", "127.0.0.1").getValue
```

Comment:
It is recommended that the configuration be moved to the package org.apache.linkis.server.conf.SessionHAConfiguration